test/Microsoft.AspNetCore.Session.Tests/SessionTests.cs

```diff
@@ -105,8 +105,11 @@ public class SessionTests
                {
                    app.UseSession(new SessionOptions
```

Comment:
Unrelated to this PR, but this is pretty gross, given that Session is using `IOptions<SessionOptions>` we should really kill this pattern.

If we want to avoid making them go `Configure<SessionOptions>(o => )`, we can change `UseSession(o => o.Cookie.Name = "TestCookie"), this new SessionOptions/Wrapper pattern stuff needs to burn....